nemo_text_processing/g2p/modules.py

```diff
@@ -379,31 +369,48 @@ def _parse_file_by_lines(p):
         with open(p, 'r') as f:
             return [l.rstrip() for l in f.readlines()]
 
-    def _parse_as_cmu_dict(self, phoneme_dict_path):
-        """Loads IPA CMUdict, and generates a set of all valid symbols."""
+    def _parse_as_cmu_dict(self, phoneme_dict_obj: dict) -> Tuple[DefaultDict[str, List], set]:
```

Comment:
We can rename this to remove the cmudict reference.

Reply:
agreed!

Reply:
will change as `_normalize_dict()`.